util/sdkutil/string.go

```diff
@@ -30,3 +31,13 @@ func FormatDec(d sdk.Dec) string {
 func FormatDecCoin(c sdk.DecCoin) string {
 	return fmt.Sprintf("%s %s", FormatDec(c.Amount), c.Denom)
 }
+
+func GenerateString(length uint) string {
```

Comment:
Since this function is only used for tests, then it should go to `tests/tsdk` to not pollute this package.